openedx/features/badging/tests/test_helpers.py

```diff
@@ -279,3 +275,17 @@ def test_filter_earned_badge_by_joined_team_course_with_no_team(self):
 
         self.assertIsNone(course_team)
         self.assertEqual(earned_badges, list())
+
+    @mock.patch('openedx.features.badging.helpers.publish_notification_to_user')
+    def test_send_user_badge_notification(self, publish_notification_to_user):
+        """
+        Test badge notification is sent to user
+        :param publish_notification_to_user: mock send notification through edx notifications
+        :return: None
+        """
+        # register badge notification type
+        badge_handlers.register_notification_types(None)
+        newly_earned_badge = BadgeFactory()
+
+        badge_helpers.send_user_badge_notification(self.user, 'dummy_badge_url', newly_earned_badge.name)
```

Comment:
Lets update code, `newly_earned_badge = BadgeFactory()` is redundant, and can he removed (update imports as well)

` badge_helpers.send_user_badge_notification(self.user, 'dummy_badge_url', 'any_badge_name')`

Reply:
done